requests/models.py

```diff
@@ -81,7 +81,7 @@ def _encode_params(data):
         """
 
         if isinstance(data, (str, bytes)):
-            return data
+            return to_native_string(data)
         elif hasattr(data, 'read'):
             return data
```

Comment:
You don't. It's a file-like object. Calling `to_native_string` on it is absolutely the wrong thing. That's why @Lukasa merged this. :)